tests/server/test_resource_list.py

```diff
@@ -449,6 +450,43 @@ async def test_resources_paging(server, client, order_by_column, order, env_with
     assert response["metadata"] == {"total": 5, "before": 2, "after": 1, "page_size": 2}
 
 
+async def test_none_resources_paging(server, client, env_with_resources):
+    """Test querying resources with paging, using different sorting parameters."""
+    env = env_with_resources
+    order_by_column = "agent"
+    order = "DESC"
+
+    no_result = await client.resource_list(
+        env.id,
+        limit=2,
+        sort=f"{order_by_column}.{order}",
+        start="ausf",
+        first_id="anet_co%3A%3Aresource%3A%3AResource%5BINMANTA-INMANTA-INMT%2Cresource_id%3Dinmt%5D",
```

Comment:
Apart from the confidentiality thing, which has been resolved, is there any reason we use so complex (difficult to read) resource here, rather than something simple and obviously meaningless like you have in the other test case with "zzz"?